<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Customers</class>
 <widget class="QDialog" name="Customers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>435</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>699</width>
    <height>435</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>699</width>
    <height>435</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AvisionR - Clients</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(255, 255, 255);
background-color: #262626;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_Last">
     <property name="styleSheet">
      <string notr="true">border: 1px solid #686868;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEdit_First">
     <property name="styleSheet">
      <string notr="true">border: 1px solid #686868;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="lineEdit_Phone">
     <property name="styleSheet">
      <string notr="true">border: 1px solid #686868;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QLineEdit" name="lineEdit_Account">
     <property name="styleSheet">
      <string notr="true">border: 1px solid #686868;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QTableView" name="tableView_Customers">
     <property name="styleSheet">
      <string notr="true">QHeaderView::section {
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 #616161, stop: 0.5 #505050,
                                       stop: 0.6 #434343, stop:1 #656565);
     color: white;
     padding-left: 4px;
     border: 1px solid #6c6c6c;

}

QTableCornerButton::section 
{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 #616161, stop: 0.5 #505050,
                                       stop: 0.6 #434343, stop:1 #656565);
     color: white;
     padding-left: 4px;
     border: 1px solid #6c6c6c;
}

color:rgb(255, 255, 255);
background-color: #262626;
</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCommandLinkButton" name="commandLinkButton_New">
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>New</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>494</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="4">
    <widget class="QCommandLinkButton" name="commandLinkButton_Search_2">
     <property name="maximumSize">
      <size>
       <width>91</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QCommandLinkButton" name="commandLinkButton_Search">
     <property name="maximumSize">
      <size>
       <width>91</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_Account</sender>
   <signal>returnPressed()</signal>
   <receiver>commandLinkButton_Search</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>623</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>644</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_First</sender>
   <signal>returnPressed()</signal>
   <receiver>commandLinkButton_Search</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>644</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Last</sender>
   <signal>returnPressed()</signal>
   <receiver>commandLinkButton_Search</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>644</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Phone</sender>
   <signal>returnPressed()</signal>
   <receiver>commandLinkButton_Search</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>644</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
